File: UI/ContactUI/src/models/email.ts
import { IEmail } from "../Interfaces/iemail";

export class Email implements IEmail {
  public id: string;
  public personId: string;
  public address: string;

  constructor (id:string,personId:string,address:string){
    this.id = id;
    this.personId = personId;
    this.address = address;
  }

  public static CreateEmpty():Email{
    return new Email('','','');
  }

}

File: Contact.Tests/TestDTOs/Email.cs
using System;
using Contact.Glue.Interfaces.DTOs;

namespace Contact.Tests.TestDTOs
{
    internal class Email : IEmail
    {
        public Guid EmailId { get; set; }
        public Guid PersonId { get; set; }
        public string Address { get; set; }
        public bool Deleted { get; set; }
        public bool DefaultEmail { get; set; }
    }
}